learn/getting_started/quick_start.md

```diff
@@ -53,11 +53,13 @@ These commands launch the **latest stable release** of Meilisearch.
 # Fetch the latest version of Meilisearch image from DockerHub
 docker pull getmeili/meilisearch:latest
 
-# Launch Meilisearch
+# Launch Meilisearch with a master key and deactivate analytics
 docker run -it --rm \
     -p 7700:7700 \
+    -e MEILI_MASTER_KEY='myMasterKey'\
     -v $(pwd)/meili_data:/meili_data \
     getmeili/meilisearch:latest
+    /meilisearch --no-analytics
```

Comment:
I didn't follow the issue very closely during v0.27, but I thought [our official recommendation](https://github.com/meilisearch/meilisearch/issues/2235) was using `meilisearch` with Docker (so, no `/`)?

```suggestion
    meilisearch --no-analytics
```

Reply:
You're right!